ember-accounts/app/templates/settings/profile.hbs

```diff
@@ -12,3 +12,7 @@
     </div>
   </div>
 </div>
+
+<div class="question-label">
+  <strong>Have questions?</strong> Email us anytime at <a href='mailto:support@huboard.com'> support@huboard.com </a>
```

Comment:
Which whitespace are you pointing out? The ws before and following `support@huboard.com` ?